plugin-modernizer-core/src/test/java/io/jenkins/tools/pluginmodernizer/core/model/JDKTest.java

```diff
@@ -148,9 +148,9 @@ public void getBuildableJdk() {
         assertEquals(JDK.JAVA_17, JDK.get("2.463").get(0));
         assertEquals(JDK.JAVA_21, JDK.get("2.463").get(1));
 
-        assertEquals(2, JDK.get("2.479.1").size());
-        assertEquals(JDK.JAVA_17, JDK.get("2.479.1").get(0));
-        assertEquals(JDK.JAVA_21, JDK.get("2.479.1").get(1));
+        assertEquals(2, JDK.get("2.479.3").size());
```

Comment:
Not needed. Also 2.479.1 is the first LTS require java 17. So I would not change on JDK info.